decidim-elections/app/cells/decidim/elections/election_m_cell.rb

```diff
@@ -17,6 +17,14 @@ def title
         present(model).title
       end
 
+      def has_image?
+        model.attachments.first.present? && model.attachments.first.file.content_type.start_with?("image")
+      end
+
+      def resource_image_path
+        model.attachments.first.url if has_image?
```

Comment:
Similar to previous comment, you could use the `:photos`:

```suggestion
        model.attachments.photos.first.url if has_image?
```